tests/test_partitions.py

```diff
@@ -0,0 +1,50 @@
+import re
+from collections import defaultdict
+from collections import OrderedDict
+import sys
+sys.path.append('../user-query-optimizer')
+import optimizer
+import sqlparse
+
+def test_partitions(queries, op):
+    # dictionary from test-query-file -> list of line numbers in that query with an approx optimization
+    correct_ops = {
+        'test-query-1.txt': [0],
+        'test-query-2.txt': [10, 17, 22, 35],
+        'test-query-3.txt': [0],
+        'test-query-4.txt': [0, 4],
+        'test-query-5.txt': [3],
+        'test-query-6.txt': [0], # for some reason, sqlparse not recognizing two statemnets
```

Comment:
This seems concerning, can you expand on what the problem is?

Reply:
Yeah, usually when there's more than one query in a file, sqlparse should be able to split it into more than one statement, but it's not doing that for this file. I don't think this would be a huge problem for the optimizer, because it'd only be given one query at a time anyways?

Reply:
Agreed, not a problem here, but can you file an issue with sqlparse?

Reply:
Alright!